tests/app/celery/test_tasks.py

```diff
@@ -545,6 +546,7 @@ def test_should_save_sms_if_restricted_service_and_valid_number(
     notification_id = uuid4()
     encrypt_notification = encryption.encrypt(notification)
 
+    # Intermittently makes the status 'technical-failure'
```

Comment:
Does this comment mean something intentional or does this indicate an intermittent problem?

Reply:
There are a lot marked serial for this reason, many of them having to do with one of the nested methods: persist_notification. We have a [pt1 ticket for cleaning up serial tests](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/gh/department-of-veterans-affairs/notification-api/1690), this would fall into a pt2 or pt3. 

When we made the #1436  change Dave and I agreed it would be best to mark any of these that popped up as serial and deal with them in the future, annotating where the problem was likely happening.